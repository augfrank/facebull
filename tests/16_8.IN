M0   C0   C1  2904
M1   C0   C2  2918
M2   C0   C3  2076
M3   C0   C4  2273
M4   C0   C5  2886
M5   C0   C6  2210
M6   C0   C7  2728
M7   C0   C8  2490
M8   C0   C9  2970
M9   C0   C10  2102
M10   C0   C12  2994
M11   C0   C13  2320
M12   C0   C14  2047
M13   C0   C15  2105
M14   C1   C0  2811
M15   C1   C2  2811
M16   C1   C3  2705
M17   C1   C4  2004
M18   C1   C6  2139
M19   C1   C7  2078
M20   C1   C8  2323
M21   C1   C9  2480
M22   C1   C10  2391
M23   C1   C11  2570
M24   C1   C12  2233
M25   C1   C13  2373
M26   C1   C14  2597
M27   C1   C15  2734
M28   C2   C0  2675
M29   C2   C1  2896
M30   C2   C3  2845
M31   C2   C4  2334
M32   C2   C5  2814
M33   C2   C6  2922
M34   C2   C7  2909
M35   C2   C8  2767
M36   C2   C9  2901
M37   C2   C10  2757
M38   C2   C11  2047
M39   C2   C12  2283
M40   C2   C13  2289
M41   C2   C14  2329
M42   C2   C15  2143
M43   C3   C0  2091
M44   C3   C1  2001
M45   C3   C4  2824
M46   C3   C5  2573
M47   C3   C8  2373
M48   C3   C9  2210
M49   C3   C10  2569
M50   C3   C11  2825
M51   C3   C12  2611
M52   C3   C14  2128
M53   C3   C15  2865
M54   C4   C0  2370
M55   C4   C2  2508
M56   C4   C3  2786
M57   C4   C6  2131
M58   C4   C7  2308
M59   C4   C8  2504
M60   C4   C9  2323
M61   C4   C10  2226
M62   C4   C11  2125
M63   C4   C12  2663
M64   C4   C13  2943
M65   C4   C14  2907
M66   C5   C0  2464
M67   C5   C1  2227
M68   C5   C3  2606
M69   C5   C4  2461
M70   C5   C6  2997
M71   C5   C7  2999
M72   C5   C8  2327
M73   C5   C9  2774
M74   C5   C11  2578
M75   C5   C12  2881
M76   C5   C13  2158
M77   C5   C14  2239
M78   C6   C0  2495
M79   C6   C1  2493
M80   C6   C2  2018
M81   C6   C3  2906
M82   C6   C4  2243
M83   C6   C5  2626
M84   C6   C7  2275
M85   C6   C8  2828
M86   C6   C9  2016
M87   C6   C11  2225
M88   C6   C12  2035
M89   C6   C13  2020
M90   C6   C14  2138
M91   C6   C15  2309
M92   C7   C0  2502
M93   C7   C1  2663
M94   C7   C2  2877
M95   C7   C3  2186
M96   C7   C5  2574
M97   C7   C6  2036
M98   C7   C8  2414
M99   C7   C10  2738
M100   C7   C11  2116
M101   C7   C12  2243
M102   C7   C14  2313
M103   C7   C15  2797
M104   C8   C0  2614
M105   C8   C1  2342
M106   C8   C2  2556
M107   C8   C3  2757
M108   C8   C4  2970
M109   C8   C5  2247
M110   C8   C6  2975
M111   C8   C7  2655
M112   C8   C10  2732
M113   C8   C11  2250
M114   C8   C13  2391
M115   C8   C14  2779
M116   C8   C15  2313
M117   C9   C0  2521
M118   C9   C1  2228
M119   C9   C2  2731
M120   C9   C3  2863
M121   C9   C4  2243
M122   C9   C5  2944
M123   C9   C6  2428
M124   C9   C7  2587
M125   C9   C8  2977
M126   C9   C10  2608
M127   C9   C11  2719
M128   C9   C12  2149
M129   C9   C15  2394
M130   C10   C0  2488
M131   C10   C1  2999
M132   C10   C2  2831
M133   C10   C3  2111
M134   C10   C4  2309
M135   C10   C5  2588
M136   C10   C6  2897
M137   C10   C7  2922
M138   C10   C8  2429
M139   C10   C9  2571
M140   C10   C11  2470
M141   C10   C12  2161
M142   C10   C13  2869
M143   C10   C14  2083
M144   C10   C15  2733
M145   C11   C0  2869
M146   C11   C1  2972
M147   C11   C2  2066
M148   C11   C3  2466
M149   C11   C5  2416
M150   C11   C6  2440
M151   C11   C7  2520
M152   C11   C8  2729
M153   C11   C9  2495
M154   C11   C10  2004
M155   C11   C12  2198
M156   C11   C13  2861
M157   C11   C14  2444
M158   C11   C15  2175
M159   C12   C0  2189
M160   C12   C1  2283
M161   C12   C3  2655
M162   C12   C4  2823
M163   C12   C5  2656
M164   C12   C6  2025
M165   C12   C7  2477
M166   C12   C8  2279
M167   C12   C9  2676
M168   C12   C13  2615
M169   C12   C14  2672
M170   C13   C0  2308
M171   C13   C1  2010
M172   C13   C2  2973
M173   C13   C3  2855
M174   C13   C4  2050
M175   C13   C5  2629
M176   C13   C6  2004
M177   C13   C7  2518
M178   C13   C8  2869
M179   C13   C9  2803
M180   C13   C10  2745
M181   C13   C11  2068
M182   C13   C12  2424
M183   C13   C14  2279
M184   C13   C15  2056
M185   C14   C0  2117
M186   C14   C1  2667
M187   C14   C2  2498
M188   C14   C3  2790
M189   C14   C4  2136
M190   C14   C5  2385
M191   C14   C6  2194
M192   C14   C7  2018
M193   C14   C8  2718
M194   C14   C9  2872
M195   C14   C10  2313
M196   C14   C11  2576
M197   C14   C12  2713
M198   C14   C13  2389
M199   C14   C15  2991
M200   C15   C0  2565
M201   C15   C1  2074
M202   C15   C2  2098
M203   C15   C3  2144
M204   C15   C4  2490
M205   C15   C5  2880
M206   C15   C6  2293
M207   C15   C7  2504
M208   C15   C8  2654
M209   C15   C9  2364
M210   C15   C10  2459
M211   C15   C11  2389
M212   C15   C12  2571
M213   C15   C13  2311
M214   C15   C14  2352
