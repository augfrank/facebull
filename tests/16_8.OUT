33641
12 17 38 44 54 76 86 97 109 128 133 154 166 184 192 202
